content/docs/command-reference/gc.md

```diff
@@ -92,7 +92,9 @@ The default remote is cleaned (see `dvc config core.remote`) unless the
 - `-j <number>`, `--jobs <number>` - parallelism level for DVC to access data
   from remote storage. This only applies when the `--cloud` option is used, or a
   `--remote` is given. The default value is `4 * cpu_count()`. For SSH remotes,
-  the default is `4`. Using more jobs may improve the overall transfer speed.
+  the default is `4`. Note that the default value can be set using the `jobs`
+  config option with `dvc remote modify`. Using more jobs may improve the
+  overall transfer speed.
```

Comment:
`gc` is not about transfer, @efiop do we parallelize delete operations in `gc`?

Reply:
We don't. Same as with status, this might speedup checking files on remotes. Could just delete the last sentence.

Reply:
> Note that that sentence was already there so maybe out of scope. I can address this separately if needed, pls lmk.

Reply:
> gc is not about transfer

Changed to "connection speed" everywhere for now.

Reply:
Sorry, not everywhere. In gc and status for now.